src/app/container/version-modal/version-modal.component.html

```diff
@@ -355,31 +355,31 @@ <h4 mat-dialog-title>{{ TagEditorMode[mode] }} Version Tag</h4>
               </div>
             </div>
           </div>
-          <div class="form-group form-group-sm" *ngIf="!unsavedVersion.create">
```

Comment:
There is no `create` property on `unsavedVersion`. If it doesn't exist, then the `*ngIf` always evaluates to true, so just removed the `*ngIf` entirely.

I'm a little wary of this change; I wonder if it was from the previous way Angular used to handle forms? But I tried invoking the UI manually, verified in the debugger that the property doesn't exist, and it seemed to work fine.